docs/guides/estimate-job-run-time.mdx

```diff
@@ -122,6 +122,22 @@ Output:
 {'quantum_seconds': 4.1058720028432445}
 ```
 </LegacyContent>
+<CloudContent>
+
+## Estimate usage before submitting a job
+
+While getting an accurate local estimation is complicated by the extra operations done for error suppression and mitigation, you can use this baseline formula to get an approximation of estimated usage:
+
+`<per sub-job overhead> + (rep_delay + <circuit length>) * <num executions>`
+
+- `<per sub-job overhead>` is a fixed overhead of 2s per sub-job. This includes operations such as loading the payload into control electronics. Your primitive job may be divided into multiple sub-jobs if it is too large for the execution engine to process all at once. 
+- `rep_delay` is a [user-customizable](/api/qiskit-ibm-runtime/options-execution-options-v2#rep_delay) option, and the default is given by `backend.default_rep_delay`, which is 250 microseconds on most IBM Quantum backends.
```

Comment:
We should link to the [page](https://docs.quantum.ibm.com/guides/repetition-rate-execution) that talks about the trade off of setting a smaller `rep_delay`, so people don't think it's an easy way to cut cost.